### ISSUE TITLE
segmentation fault

### ISSUE THREAD

**Issue by jeekajoo** [ozmartian/vidcutter#46]
Hi,
I'm using vidcutter `3.2.0~201705111415~ubuntu16.04.1` and I'm experiencing segmentation faults. Especially when browsing using multiple 5 seconds forwards (Alt + up).
EDIT: It can also happen when saving media.

Here is a sample video on which I'm able to reproduce it everytime: https://pub.jeekajoo.eu/sample.mp4
So that you can try by yourself.

Debug log:
```
minimum size set to: PyQt5.QtCore.QSize(900, 640) (NORMAL)
2017-05-14 16:42 - vidcutter.libs.videoservice - INFO - VideoService: backend = "/usr/bin/ffmpeg"	mediainfo = "/usr/bin/mediainfo"
2017-05-14 16:42 - vidcutter.videocutter - INFO - MPV info - cplayer: Playing: /home/jeekajoo/gopro/sample.mp4
2017-05-14 16:42 - vidcutter.videocutter - INFO - MPV info - cplayer:  (+) Video --vid=1 (*) (h264)
2017-05-14 16:42 - vidcutter.videocutter - INFO - MPV info - cplayer:  (+) Audio --aid=1 --alang=eng (*) (aac)
libva info: VA-API version 0.39.0
libva info: va_getDriverName() returns 0
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/i965_drv_video.so
libva info: Found init function __vaDriverInit_0_39
libva info: va_openDriver() returns 0
2017-05-14 16:42 - vidcutter.videocutter - INFO - MPV info - cplayer: AO: [pulse] 48000Hz stereo 2ch float
2017-05-14 16:42 - vidcutter.videocutter - INFO - MPV info - vd: Using hardware decoding (vaapi).
2017-05-14 16:42 - vidcutter.videocutter - INFO - MPV info - cplayer: VO: [opengl] 1920x1080 vaapi
[1]    1040 segmentation fault  vidcutter --debug
```
Thanks

**Comment by PoulKalff** [comment]
segmentation fault also often happens when holding down KEYLEFT or KEYRIGHT

**Comment by jeekajoo** [comment]
@ozmartian Looks like you fixed it with `3.5.0~201705311838~ubuntu16.04.1` 

Thank you
